core/app/modules/kit-library/assets/js/components/item-header.js

```diff
@@ -39,6 +39,19 @@ function useKitCallToActionButton( model, { apply, isApplyLoading, onConnect } )
 		}
 
 		if ( type === TYPE_PROMOTION && subscriptionPlan ) {
+			const buildSubscriptionLink = () => {
+				let url = subscriptionPlan.promotion_url;
+				if ( model.title ) {
+					// Remove special characters, replace spaces with '-' and convert url kit name to lowercase.
+					let cleanTitle = model.title.replace( /[^\w\s]/g, '' ).replace( /\s/g, '-' ).toLowerCase();
```

Comment:
- Should use `const` instead.
- What if the title already includes a dash? Would not it be better to switch the replaces?
`const cleanTitle = model.title.replace( /\s/g, '-' ).replace( /[^\w\s-]/g, '' ).toLowerCase();`